repositories/PageRepository.test.js

```diff
@@ -494,13 +496,33 @@ describe("PagesRepository", () => {
 
       const duplicatePage = await PageRepository.duplicatePage(result.id, 1);
 
-      const fieldsToOmit = ["id", "order", "title", "createdAt", "updatedAt"];
+      const fieldsToOmit = [
+        "id",
+        "order",
+        "alias",
+        "title",
+        "createdAt",
+        "updatedAt"
+      ];
 
       expect(omit(result, fieldsToOmit)).toMatchObject(
         omit(duplicatePage, fieldsToOmit)
       );
     });
 
+    it("prepends 'Copy of' to the question alias of the copy when duplicating a page", async () => {
+      const { section } = await setup();
+
+      const page = buildPage({ sectionId: section.id });
+      const result = await PageRepository.insert(page);
+
+      const duplicatePage = await PageRepository.duplicatePage(result.id, 1);
```

Comment:
Isn't this all handled in the DuplicateStrategy?

Reply:
Ah, I see it calls it internally. But surely it should handle the name thing as well?

Reply:
What name thing?

This test asserts `alias` is appended with `Copy of`.

There is a test below which as asserts `title` is appended with `Copy of`.

Reply:
I'm questioning - why is it happening here. AFAIK the Strategy should handle that for you

Reply:
I agree. This implementation detail should *really* be within `duplicatePageStrategy` as this method is so specific and has no other use case.  Therefore, should handle everything about duplicating a page.  

Do we want to do this in this PR, a separate PR or maybe when we tackle this refactor during the duplicate Sections feature?

Reply:
I think the logic is already in the Strategy - so you could call it with just a  page and it might do it for you.

Reply:
As far as I can tell, there is no logic to handle prefixing in the strategy.